scss/_variables.scss

```diff
@@ -787,6 +794,8 @@ $breadcrumb-padding-y:          .75rem !default;
 $breadcrumb-padding-x:          1rem !default;
 $breadcrumb-item-padding:       .5rem !default;
 
+$breadcrumb-margin:             0 0 1rem !default;
```

Comment:
Crap, I typo-ed—I meant shouldn't be here. It's in the wrong PR, so you can remove it.